docs/explanations/architecture/styles.md

```diff
@@ -498,3 +496,68 @@ Similarly to block supports, there can be only one instance of any style in use
 4. **Only blocks using block supports are shown in the Global Styles UI**
 
 The global styles UI in the site editor has a screen for per-block styles. The list of blocks is generated dynamically using the block supports from the `block.json` of blocks. If a block wants to be listed there, it needs to use the block supports mechanism.
+
+### Structural layout styles
+
+In addition to styles at the individual block level and in global styles, there is the concept of Layout styles that are output for both blocks-based and classic themes.
+
+The Layout block support is an experimental approach for outputting common structural layout styles that are shared between blocks that are used for creating layouts. Layout styles are useful for providing common styling for any block that is a container for other blocks. Examples of blocks that depend on these layout styles include Group, Row, Columns, Buttons, and Social Icons.
+
+While the feature is part of WordPress core, it is still flagged as experimental in the sense that the features and output are still undergoing active development. It is therefore not yet a stable feature from the perspective of 3rd party blocks, as the API is likely to change. The feature is enabled in core blocks via the `__experimentalLayout` setting under `supports` in a block's `block.json` file.
+
+There are two primary places where Layout styles are output:
+
+1. Base layout styles
+
+Base layout styles are those styles that are common to all blocks that opt-in to a particular layout type. Examples of common base layout styling include setting `display: flex` for blocks that use the Flex layout type (such as Buttons and Social Icons), and providing default max-width for constrained layouts.
+
+Base layout styles are output from within [the main class](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/class-wp-theme-json.php) that handles global styles, and form part of the global styles stylesheet. In order to provide support for core blocks in classic themes, these styles are always output, irrespective of whether the theme provides its own `theme.json` file.
+
+Common layout definitions are stored in [the core `theme.json` file](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/theme.json), but are not intended to be extended or overridden by themes.
+
+2. Individual layout styles
+
+When a block that opts-in to the experimental Layout support is rendered, two things are processed and added to the output:
```

Comment:
I am being hypocritical coz I do this too, and feel free to ignore:

 "opt in/out", without hyphen is the verb. "opt-in/out", with the hyphen, is the noun or adjective.

Reply:
Ah, yes, just like `log in` versus `login` or `log-in` 👍